test/spec.js

```diff
@@ -432,6 +432,44 @@ test('it should not return any Entity given an invalid ID', function(t) {
   .expect(404, pass(t, 'returned 404 not found'));
 });
 
+test('it should allow metadata properties to be posted and returned', function(t) {
+  const one = genEntity();
+
+  request(server)
+  .post('/entities')
+  .auth('test', key)
+  .send(one)
+  .expect(200)
+  .end((err, oneRes) => {
+    const perms = {
+      type: 'bar',
+      entity: oneRes.body.id,
+      metadata: {
+        type: "little creatures"
+      }
+    };
+
+    var two = genEntity();
+    two.permissions = [perms];
```

Comment:
No need to declare this as a var, by the way. const would work just as well in this case. The const declaration type in JS doesn't prevent you from assigning to a property.